browser/src/Services/Themes/ThemeManager.ts

```diff
@@ -269,11 +368,78 @@ export const DefaultThemeColors: IThemeColors = {
     "fileExplorer.selection.foreground": HighlightForeground,
     "fileExplorer.cursor.background": NormalMode,
     "fileExplorer.cursor.foreground": NormalMode,
+
+    "editor.tokenColors": {
+        "variable.object": {
+            settings: "identifier",
+            scope: "variable.object",
+            color: null,
+        },
+        "variable.other.constant": {
+            settings: "constant",
+            scope: "variable.other.constant",
+            color: null,
+        },
+        "variable.language": {
+            settings: "identifier",
+            scope: "variable.language",
+            color: null,
+        },
+        "variable.parameter": {
+            settings: "identifier",
+            scope: "variable.parameter",
+            color: null,
+        },
+        "variable.other": {
+            settings: "identifier",
+            scope: "variable.other",
+            color: null,
+        },
+        "support.function": {
+            settings: "function",
+            scope: "support.function",
+            color: null,
+        },
+        "entity.name": {
+            settings: "function",
+            scope: "entity.name",
+            color: null,
+        },
+        "entity.other": {
+            settings: "constant",
+            scope: "entity.other",
+            color: null,
+        },
+        identifier: {
+            color: null,
+        },
+        function: {
+            color: null,
+        },
+        constant: {
+            color: null,
+        },
+    },
+}
+
+export interface IVimHighlight {
```

Comment:
Ideally, we could make the ThemeManager agnostic of Vim. The `highlight group` is really an implementation detail - it'd be great if we could make this care about color/bold/italic/background-color, and then leave it up to the `setHighlights` in `Buffer` to rationalize highlight groups from that. It sounded like there might be challenges here though - let me know what sort of hurdles you ran into.